src/data/linux-quiz.ts

```diff
@@ -1,5 +1,17 @@
 const linuxQuiz = [
   {
+    
+    Question: "Say a file named 'list' has has intial file permissions as '-rwxrwxrwx', A person wants to change its permissions using 'chmod' such that GROUP and OTHER can only read it, neither write or execute it?",
```

Comment:
This question is a little to long. Let's shorten it. 
```suggestion
    Question: "In Linux, which command is used to remove the writing and executing permissions for 'group' and 'other'?",
```